bundle/edu.gemini.horizons.api/src/main/java/edu/gemini/horizons/server/backend/CgiQueryExecutor.java

```diff
@@ -94,10 +94,6 @@ public HorizonsReply execute(HorizonsQuery query) throws HorizonsException {
             client.executeMethod(method);
             reply = CgiReplyBuilder.buildResponse(method.getResponseBodyAsStream(), method.getRequestCharSet());
             method.releaseConnection();
-        } catch (HorizonsException ex) {
-            throw ex;
-        } catch (HttpException ex) {
-            throw HorizonsException.create(ex);
         } catch (IOException ex) {
             throw HorizonsException.create(ex);
```

Comment:
Note that since `HorizonException` is just passed through, there is no need to include it in the `try`-`catch` block.
Aslo, `HttpException`s are `IOException`s, so no special handling is needed of these since they are treated identically to `IOException`s.